.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: test
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: Build charm
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        set -eux
+        sudo snap install charmcraft --classic
+        sudo snap install juju --classic
+        sudo snap install juju-wait --classic
+
+    - name: Init LXD
+      run: |
+        set -eux
+        sudo adduser "$USER" lxd
+        sudo -g lxd lxd init --auto
+
+    - name: Build charm
+      run: |
+        set -eux
+        sudo -g lxd charmcraft build
+
+    - name: Bootstrap Juju
+      run: |
+        set -eux
+        sudo -g lxd lxc network set lxdbr0 ipv6.address=none
```

Comment:
Despite some documentation still suggesting it, I'm not sure it's needed anymore.

Reply:
It is, unfortunately.